pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -661,7 +661,12 @@ func (cm *containerManagerImpl) GetResources(pod *v1.Pod, container *v1.Containe
 		if err != nil {
 			return nil, err
 		}
+		// NOTE: Passing CDI device names as annotations is a temporary solution
+		// It will be removed after all runtimes are updated
+		// to get CDI device names from the ContainerConfig.CDIDevices field
 		opts.Annotations = append(opts.Annotations, resOpts.Annotations...)
+
```

Comment:
nit: I would remove this line unless the linter is telling you to put it there for some reason.